executor/builder.go

```diff
@@ -4718,6 +4760,21 @@ func (b *executorBuilder) buildBatchPointGet(plan *plannercore.BatchPointGetPlan
 	return e
 }
 
+func newReplicaReadAdjuster(ctx sessionctx.Context, avgRowSize float64) txnkv.ReplicaReadAdjuster {
+	return func(count int) (tikv.StoreSelectorOption, clientkv.ReplicaReadType) {
+		if int64(avgRowSize*float64(count)) >= ctx.GetSessionVars().AdaptiveClosestReadThreshold {
```

Comment:
I do not understand what this check mean, why use AdaptiveClosestReadThreshold to decide whether to use ReplicaReadMixed?